portfolio/src/main/webapp/style/format.css

```diff
@@ -299,4 +291,92 @@ a{
 	color: #fff;
 	transform: scale(1.1);
 	border: #5cb85c 1px solid;
+}
+
+.comments-section{
```

Comment:
nit: add space before brace

let's run the code formatter again to be sure

Reply:
done